notebooks/file_calibration.ipynb

```diff
@@ -574,10 +574,16 @@
    },
    "outputs": [],
    "source": [
-    "def array_class_plot(cls):\n",
-    "    fig, axes = plt.subplots(1, 2, figsize=(14, 6), dpi=100, gridspec_kw={'width_ratios': [2, 1]})\n",
-    "    plot_antclass(hd.antpos, cls, ax=axes[0], ants=[ant for ant in hd.data_ants if ant < 320], legend=False, title='HERA Core')\n",
-    "    plot_antclass(hd.antpos, cls, ax=axes[1], ants=[ant for ant in hd.data_ants if ant >= 320], radius=50, title='Outriggers')"
+    "def array_class_plot(cls, extra_label=\"\"):\n",
+    "    outriggers = [ant for ant in hd.data_ants if ant >= 320]\n",
+    "\n",
+    "    if outriggers:\n",
+    "        fig, axes = plt.subplots(1, 2, figsize=(14, 6), dpi=100, gridspec_kw={'width_ratios': [2, 1]})\n",
+    "        plot_antclass(hd.antpos, cls, ax=axes[0], ants=[ant for ant in hd.data_ants if ant < 320], legend=False, title=f'HERA Core{extra_label}')\n",
+    "        plot_antclass(hd.antpos, cls, ax=axes[1], ants=outriggers, radius=50, title='Outriggers')\n",
+    "    else:\n",
+    "        fig, axes = plt.subplots(1, 1, figsize=(14, 6), dpi=100)\n",
```

Comment:
This aspect ratio will look very silly for a single panel.

Reply:
Changed down to (9,6) which should give a similar aspect as the double-panel.